pt/news/_posts/2016-09-08-ruby-2-4-0-preview2-released.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: news_post
+title: "Lançado Ruby 2.4.0-preview2"
+author: "naruse"
+translator: "jcserracampos"
+date: 2016-09-08 18:00:00 +0900
+lang: pt
+---
+
+Estamos satisfeitos em anunciar o lançamento do Ruby 2.4.0-preview2.
+
+Ruby 2.4.0-preview2 é o segundo *preview* do Ruby 2.4.0.
+Este preview2 está sendo lançado para receber comentários da comunidade.
+Sinta-se a vontade para
+[enviar comentários](https://bugs.ruby-lang.org/projects/ruby/wiki/HowToReport)
+sendo que você ainda pode influenciar as funcionalidades.
```

Comment:
I think `mudar` or `alterar` sounds better here instead of `influenciar`, just like your other PR #1470.